core/src/main/kotlin/org/jetbrains/research/testspark/core/data/TestGenerationData.kt

```diff
@@ -23,14 +21,10 @@ data class TestGenerationData(
     // changing parameters with a large prompt
     var polyDepthReducing: Int = 0,
     var inputParamsDepthReducing: Int = 0,
-
-    // list of correct test cases during the incorrect compilation
-    val compilableTestCases: MutableSet<TestCaseGeneratedByLLM> = mutableSetOf(),
```

Comment:
I removed this variable because we never used its value (only wrote in it); test cases that are rendered in the sidebar are taken from `Report.testCaseList` variable.